examples/Receiver_Response.py

```diff
@@ -0,0 +1,164 @@
+""" Calculate and Fit Receiver Response to Sinc
+===============================================
+
+Two files are required for the following example:
+
+    File1 contains the analytic signal acquired on the GDS oscilloscope
+    directly output from the AFG output. Each node pertains to signal with a
+    different frequency (in kHz) which are fit to a complex function to extract
+    the power (in :math:`V^{2}`).
+    File2 contains the quadrature signal acquired on the receiver when the same
+    signal of File1 is injected into it. Each node pertains to signal with a
+    different frequency (in kHz) which are converted to a PSD. The power (in
+    :math:`dg^{2}`) is calculated from the peak of the convolved PSD.
+
+    The receiver response is then the ratio of :math:`dg^{2}(\\nu)` to
+    :math:`V^{2}(\\nu)` which is fit to a sinc function. The final plot shows
+    the square of the data with the fit.
+"""
+import numpy as np
+import pyspecdata as psd
+from sympy import symbols
+import sympy as sp
+import re
+
+
+def for_plot(thisdata):
+    """Converts :math:`dg^{2}/V^{2}` to :math:`dg/\\mu V`
+    and sets labels accordingly for plotting"""
+    # Take sqrt for units of $dg/V$
+    thisdata.data = np.sqrt(thisdata.data)
+    # Units of $dg/\mu V$
+    thisdata /= 1e6
+    thisdata.rename("nu_test", r"$\Delta \nu$")
+    thisdata.name(r"$\mathrm{dg_{%s\ \mathrm{kHz}}}/ \mu V$" % SW)
+    return thisdata
+
+
+lambda_G = 0.4e3  # Width for Gaussian convolution
+data_dir = "ODNP_NMR_comp/noise_tests"
+file1 = "240123_10mV_AFG_GDS_5mV_100MSPS_analytic.h5"
+file2 = "240117_afg_sc_10mV_3p9kHz_zoom.h5"
+# There are less nodes acquired for the control case (since we assume it's
+# relatively flat) so I need to separately define them
+file1_nodes = psd.find_file(
+    re.escape(file1),
+    exp_type=data_dir,
+    return_list=True,
+)
```

Comment:
for after 48, here is an example that sorts node names so that they come in frequency order.   The int is key.
```
In [17]: a = ['bla_1','bla_10','bla_3','bla_2']

In [18]: sorted(a, key=lambda x: int(x.split('_')[1]))
Out[18]: ['bla_1', 'bla_2', 'bla_3', 'bla_10']
```
use this example to write code for here, and then do NOT use the `.sort()` elsewhere, since it's redundant.

Reply:
see function above